janus_server/src/trace.rs

```diff
@@ -50,6 +67,19 @@ pub struct TokioConsoleConfiguration {
     pub listen_address: Option<SocketAddr>,
 }
 
+/// Configuration options specific to the OpenTelemetry OTLP exporter.
+#[derive(Clone, Debug, Default, PartialEq, Eq, Serialize, Deserialize)]
+pub struct OtlpConfiguration {
+    /// Enable the OpenTelemetry tracing layer, with the OTLP exporter.
+    #[serde(default)]
+    pub enabled: bool,
+    /// gRPC endpoint for OTLP exporter.
+    pub endpoint: String,
+    /// Additional metadata/HTTP headers to be sent with OTLP requests.
+    #[serde(default)]
+    pub metadata: HashMap<String, String>,
```

Comment:
IIUC the expectation is we'll put a Honeycomb API key into this metadata hashmap via the `aggregator` YAML config file. I don't think we should put a secret value like that in config. Can we plumb it through the aggregator binary's arguments, instead, so that it can be read from an environment variable populated from a Kubernetes secret?